django/core/checks/templates.py

```diff
@@ -10,6 +10,11 @@
     "in OPTIONS. Either remove APP_DIRS or remove the 'loaders' option.",
     id='templates.E001',
 )
+E002 = Error(
+    "'string_if_invalid' is not a string in your TEMPLATES. "
```

Comment:
I suggest "'string_if_invalid' in TEMPLATES OPTIONS must be a string (got (value) (type))." I think the part about "Change it.." isn't necessary.